notifications_utils/version.py

```diff
@@ -1,2 +1,2 @@
-__version__ = "48.1.1"
+__version__ = "48.1.2"
```

Comment:
I bumped utils to 48.2.0 for the lang tag fixes so you will need to bump it again here :)